src/configure/configure.ts

```diff
@@ -512,13 +514,13 @@ class Orchestrator {
             appropriatePipelines = remotePipelines;
         }
         const pipelineMap = this.getMapOfUniqueLabels(appropriatePipelines);
+        this.filterPipelinesByResourceType(pipelineMap);
```

Comment:
Rather than filtering these templates, I think it would be better to call the other API which returns one template based on language + build target + deploy target.
@anuragc617 any thoughts here?